src/kernel/emulation/linux/misc/sysctl_machdep.c

```diff
@@ -51,3 +77,132 @@ sysctl_handler(handle_brand_string)
 	return 0;
 }
 
+sysctl_handler(handle_max_basic)
+{
+	setup(0);
+
+
+    __cpuid(level, eax, ebx, ecx, edx);
+
+
+	sprintf(old,"%d",eax);
```

Comment:
This (along with the other uses of `sprintf`) produces an "implicit function definition" error with Clang. Simple solution: declare it at the top of the file, like so:
```c
extern int sprintf(char* str, const char* format, ...);
```

Reply:
No, you cannot use libc functions in syscalls. We have https://github.com/darlinghq/darling/blob/master/src/kernel/emulation/linux/simple.h for string formatting